CTP/isolation/src/com/navercorp/cubridqa/isolation/Context.java

```diff
@@ -80,7 +80,7 @@ public void reload() throws IOException {
 		this.shouldUpdateTestCase = getProperty("main.testcase.update_yn", "false").equalsIgnoreCase("true");
 		this.isWindows = getProperty("main.testing.platform", "linux").equalsIgnoreCase("windows");
```

Comment:
as is:
this.isWindows = getProperty("main.testing.platform", "linux").equalsIgnoreCase("windows");

to be:
this.isWindows = getTestPlatform().equalsIgnoreCase("windows");